src/Surfnet/StepupGateway/GatewayBundle/Entity/SamlEntity.php

```diff
@@ -110,6 +109,16 @@ public function toServiceProvider()
         $configuration['entityId']             = $this->entityId;
         $configuration['configuredLoas']       = $decodedConfiguration['loa'];
 
+        $configuration['secondFactorOnly'] = false;
+        if (isset($decodedConfiguration['second_factor_only'])) {
+            $configuration['secondFactorOnly'] = (bool) $decodedConfiguration['second_factor_only'];
```

Comment:
This is for the edge-case that the GW has been deployed, but the MW doesn't have updated projections yet.
So to make GW-SFO BWC.
Or do I tell Pieter to deploy the MW first?